praw/models/reddit/base.py

```diff
@@ -29,22 +39,35 @@ def fullname(self):
 
         """
         return '{}_{}'.format(self._reddit._objector.kind(self),
-                              self.id)  # pylint: disable=invalid-name
+                              self.id)
+
+    @property
```

Comment:
> What do you propose I do with it?

I propose we remove it. If people ask for such an attribute then we can add it, but it hasn't seemed to be an issue.

Reply:
This attribute complements `.fetch()`. It intends to give users more control over PRAW’s models.

> If people ask for such an attribute then we can add it, but it hasn't seemed to be an issue.

Nobody is going to ask for a `.fetched` attribute because its value can already be accessed via `._fetched`. Accessing underscore members isn’t recommended, of course, but we don’t have any other good way to test if a model has its attributes ready?

I’m sure if I ask around that people will agree to having a `.fetched` attribute.

If we have `.fetch()` then there should also be a `.fetched` property. I will remove `fetched` if `fetch()` gets rejected too. Otherwise, it doesn’t really make sense to have one without the other.